### PR TITLE
Species/strain query

### PR DESCRIPTION
**Pull request by jayrbolton**
A simpler and hopefully faster query to be used in species/strain autocompletes in the UI

This only searches species/strain (doc.rank == "species" OR doc.strain)

### CODE REVIEW

**Review by scanon — APPROVED** [review]
LGTM

File: stored_queries/taxonomy/taxonomy_search_species.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      title: Taxon collection name
+      examples: [ncbi_taxon, gtdb_taxon]
+    search_text:
+      type: string
+      title: Search text
+      description: Text to search on for the scientific name
+    offset:
+      type: integer
+      default: 0
+      maximum: 100000
```
Comment (MrCreosote) [inline stored_queries/taxonomy/taxonomy_search_species.yaml:18]: I don't think elastic will allow more than 10k will it?

**Review by MrCreosote — COMMENTED** [review]
How fast is the search?

File: stored_queries/taxonomy/taxonomy_search_species.yaml (hunk 1 above)
Comment (scanon) [reply stored_queries/taxonomy/taxonomy_search_species.yaml:18]: This is arango

Comment (MrCreosote) [reply stored_queries/taxonomy/taxonomy_search_species.yaml:18]: Ugh, brain fart

**Comment by jayrbolton** [comment]
I have some timings I'll post in a minute

**Comment by jayrbolton** [comment]
@MrCreosote Here is a sample of times for the old search_sci_name query, all in ms:
```
45.011
45.011
45.011
45.011
68.140
```
Here is a sample of times for this stripped down query (gives the same search results as above)
```
9.838
10.189
15.054
8.064
15.536
7.241
```
So it's definitely quicker

**Pull request merged**